CardGame/CardGameDealButton.swift

```diff
@@ -8,29 +8,30 @@
 import SwiftUI
 
 struct CardGameDealButton: View {
+    
+    private let buttonText = "DEAL"
```

Comment:
Hardcoded texts were used in multiple files. If we could store them all in a `CardGameAppConstants` file, it might be easier to maintain.